test/e2e/antreapolicy_test.go

```diff
@@ -184,7 +184,7 @@ func testMutateACNPNoRuleName(t *testing.T) {
 		SetAppliedToGroup(map[string]string{"pod": "a"}, nil, nil, nil).
 		SetPriority(10.0).
 		AddIngress(v1.ProtocolTCP, &p80, nil, nil, map[string]string{"pod": "b"}, map[string]string{"ns": "x"},
-			nil, nil, secv1alpha1.RuleActionAllow, "")
+			nil, nil, []ACNPRuleAppliedToSpec{}, secv1alpha1.RuleActionAllow, "")
```

Comment:
if you want something less verbose, you could use `nil` here instead of `[]ACNPRuleAppliedToSpec{}`

Reply:
Done.